Tasks/VsTest/task.json

```diff
@@ -29,7 +29,7 @@
   "inputs": [
     {
       "name": "testAssembly",
-      "type": "filePath",
+      "type": "string",
```

Comment:
Should it be a multi line control so you don't need the semi colons?  Of course the task would still need some compat code and support splitting on existing lines.  See the publish artifacts task which takes a list of patterns